py/desiutil/dust.py

```diff
@@ -17,6 +17,29 @@
 from .log import get_logger
 log = get_logger()
 
+def extinction_total_to_selective_ratio(band , photsys) :
+    """Return the linear coefficient R_B = A(B)/E(B-V) where A(B) = -2.5*log10(transmission in B band),
+       for band B in 'G','R' or 'Z', the optical bands of the legacy surveys. photsys = 'N' or 'S' specifies
+       the survey (BASS+MZLS or DECALS)
```

Comment:
I would recommend specifying in here what "E(B-V)" means.  I believe for DESI purposes it means E(B-V)_SFD, before any recalibrations to that quantity.  One could imagine an rv argument to this function in some future world where we have a high latitude R(V) map.

Reply:
would an extra argument ebv_source with default = "SFD" would be fine? If not please suggest an alternative.

Reply:
I clarified the comment that E(B-V) means SFD, crossing commits with Julien's comment.  OK to add ebv_source with default "SFD", but let's wait on rv option until we actually have code that could do something with it.

I also added unit tests for completeness.

Reply:
I am merging as is for now. We can go back to this in the future when we use different maps / calibrations for dust extinction.

Reply:
Sounds good.  I'm happy with just specifying that E(B-V) means SFD in the docstring.